src/Expander/TC.hs

```diff
@@ -26,10 +28,10 @@ derefType ptr =
     Just var -> pure var
 
 
-setTVKind :: MetaPtr -> VarKind Ty -> Expand ()
-setTVKind ptr k = do
+setTVLinkage :: MetaPtr -> VarLinkage Ty -> Expand ()
+setTVLinkage ptr k = do
```

Comment:
renaming "TVKind" to "TVLinkage" in order to avoid confusion between "kind" meaning "is this a type variable which is linked to another, known type expression, or is this an unlinked type variable about which we know nothing?" and "kind" meaning "the type of types".

Reply:
Good idea